modules/cdc-ext/src/test/java/org/apache/ignite/cdc/AbstractReplicationTest.java

```diff
@@ -633,6 +637,40 @@ protected String[] hostAddresses(IgniteEx[] dest) {
     /** */
     protected abstract void checkConsumerMetrics(Function<String, Long> longMetric);
 
+    /** Checks the count for CDC metrics */
+    protected abstract void checkMetricsCount(int putCnt, int rmvCnt);
+
+    /** Checks the events count for CDC metrics */
+    protected void checkMetricsEventsCount(int putCnt, int rmvCnt, Supplier<Long> supplier) {
+        try {
+            waitForCondition(() -> supplier.get() == (long)(putCnt + rmvCnt) * KEYS_CNT * (backups + 1) *
+                (mode == CacheMode.PARTITIONED ? 1 : srcCluster.length), getTestTimeout());
```

Comment:
`evtsCnt * (mode == CacheMode.PARTITIONED ? (backups + 1) : srcCluster.length)`

Reply:
Fixed this mistake